hash/c64_cass.xml

```diff
@@ -6839,14 +6720,13 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 	</software>
 
 	<software name="headovrh">
-		<description>Head Over Heels</description>
+		<description>Head Over Heels (The Hit Squad)</description>
```

Comment:
Why was "(The Hit Squad)" added here?  We only need to do this if we have a duplicate entry from a different publisher

Reply:
The one directly after it has a description of "Head Over Heels (Ocean)", so I figured either all of them need the label added to disambiguate them, or all but the first do.  If it's all but the first, then it's Ocean rather than The Hit Squad that can lose the clarification, as Ocean published it first.  (The Hit Squad being the budget label that re-released mostly Ocean games, being essentially their label.)

Reply:
Generally, the latest release (which potentially has some bug fixes/improvements) is the original version and the older versions are considered clones

Reply:
Thanks, I've taken the label out of the title for the re-release: [15f4eb4](https://github.com/mamedev/mame/pull/9307/commits/15f4eb46db0b9764a3381245fa601c532c20ee0c)  It's not really a version update so much as a budget reissue, but this is certainly more consistent.  (The people re-releasing them would often have to crack the original game, remove the loader, and add a new loader.  They didn't always have access to the original source code, so removing bugs was out of the question in these instances.)

Reply:
OK, I found [the relevant page](http://www.pauliehughes.com/freeload.htm) I'd read a while back.  The authorised cracking was more when they released compilations of non-Ocean titles, but still, the budget re-releases are more their separate thing than a "new and improved" version -- and took longer to load, due to a cheaper duplication process.

Reply:
Agreed.  They did sometimes change/add loading picture/music/mini game for the budget titles I have noticed so there are sometimes differences.  Any difference is worth preserving :-)

Reply:
Oh, definitely!  I'm a big fan of archiving both.  I just wouldn't say the budget re-releases are the definitive versions, is all, they're more their own separate series.